satzung.md

```diff
@@ -84,4 +84,6 @@ Zu den Aufgaben des Fachschaftsrates gehören insbesondere:
 
 ## § 7 Satzungsänderungen
 
-Die Änderung der Satzung kann nur erfolgen, wenn zwei Drittel der bei einer Vollversammlung stimmberechtigten Anwesenden dem Antrag zustimmen.
+(1) Die Änderung der Satzung kann nur erfolgen, wenn zwei Drittel der bei einer ordentlichen Vollversammlung stimmberechtigten Anwesenden dem Antrag zustimmen.
```

Comment:
d.h. auf einer außerordentlichen kann man nie die Satzung ändern?

Reply:
Jo, aber das ist ja auch eigentlich ne gute Sache, oder? Schließlich sind Satzungsänderungen wichtig und da sollten möglichst viele Leute anwesend/rechtzeitig informiert sein

Reply:
Ich finde das schwierig. Wenn nur der FSR ordentliche Vollversammlungen macht und so weiter. Würde das "ordentlich" streichen.

Reply:
Welche Punkte würde man denn plötzlich in einer Außerordentlichen VV ändern wollen, denen lediglich der FSR nicht zustimmen möchte? Wenn es einen solch starken Clash zwischen FSR und Studierendenschaft gibt, könnte man den FSR in einer AVV auch einfach abwählen

Reply:
@felixrindt

Reply:
Wenn du es auf ordentliche VV einschränkst bedeudet das:

* dass die Satzung nur auf VV geändert werden kann, die durch den FSR einberufen werden.
* dass die Satzung nur einmal im Jahr geändert werden kann.

Ich finde das sind komische Einschränkungen. Da geht's auch nicht darum, ob mir ein Anwendungsfall (Studiengangsanschluss) einfällt. Ich finds nur unnötig spezifisch.

Reply:
ich finde es okay, das auf ordentliche vvs zu beschränken. unter der maßgabe, dass die satzung nicht verbietet, auch mehr als eine ordentliche vv im jahr abzuhalten. ich würde nur gerne die ganzen regelungen zu längerer ankündigung und diskussionszeit im voraus dafür direkt ausnutzen.

Reply:
Wie oben geschrieben:
Für Satzungsänderungen halte ich die längere Ankündigungszeit wichtig, weshalb ich so etwas nur auf ordentlichen VVs machen würde. Deswegen würde ich es nicht auf eine ordentlich VV beschränken. Ich glaube nicht, dass es wirklich mal dazu kommt, dass sich ein FSR aus unlogischen Gründen weigert und wenn doch kann man den ja auf ner AVV abwählen.